cmd/zed/api/command.go

```diff
@@ -50,19 +53,40 @@ func DefaultHost() string {
 }
 
 func New(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
+	dir, _ := os.UserHomeDir()
+	if dir != "" {
+		dir = filepath.Join(dir, ".zapi")
```

Comment:
What do you think about changing this to `.zed`?
```suggestion
		dir = filepath.Join(dir, ".zed")
```